data/OperatingSystemView.css

```
@@ -0,0 +1,10 @@
+avatar.image {
+    background-color: alpha(black, 0.05);
```

I think a little more foreground protection would be best here, maybe

```suggestion
    background-color: alpha(black, 0.15);
```

---

we also discussed trying this in Slack

```suggestion
    background-image: linear-gradient(
        to bottom,
        alpha(@accent_color_500, 0.25),
        alpha(@accent_color_700, 0.75)
    );
```